README.md

```diff
@@ -42,9 +43,27 @@ or using CakePHP's console:
 ./bin/cake plugin load BootstrapUI
 ```
 
-## AppView Setup
-Then for a complete setup, just make your `AppView` class extends `BootstrapUI\View\UIView`.
-The `src\View\AppView.php` then will be as the following: 
+
+## Testing
```

Comment:
As I stated before info about testing should be moved near to the "Patches & Features" section or just removed. It's out of place having it in between installing and usage sections.